applications/OptimizationApplication/python_scripts/utilities/optimization_problem.py

```diff
@@ -101,6 +101,9 @@ def GetListOfControls(self) -> 'list[Control]':
     def RemoveControl(self, name: str) -> None:
         self.RemoveComponent(name, Control)
 
+    def AddProcessType(self, process_type: str) -> None:
```

Comment:
what is this function exactly ?! the name and operation do not match

Reply:
It add process routine for output.